### PR TITLE
Remove unnecessary config options from AIX

### PR DESCRIPTION
**Pull request by AdamBrousseau**
Signed-off-by: Adam Brousseau <adam.brousseau88@gmail.com>

### CODE REVIEW

**Comment by AdamBrousseau** [comment]
cc @keithc-ca 
Jenkins compile aix jdk11

File: buildenv/jenkins/variables/defaults.yml [hunk 1]
```diff
@@ -149,7 +149,7 @@ aix_ppc-64_cmprssptrs:
     8: '--with-cups-include=/opt/freeware/include --with-extra-ldflags=-lpthread --with-extra-cflags=-lpthread --with-extra-cxxflags=-lpthread DF=/usr/sysv/bin/df --disable-ccache --with-jobs=8'
     9: '--with-cups-include=/opt/freeware/include --with-extra-ldflags=-lpthread --with-extra-cflags=-lpthread --with-extra-cxxflags=-lpthread DF=/usr/sysv/bin/df --disable-warnings-as-errors --with-jobs=8'
     10: '--with-cups-include=/opt/freeware/include --with-extra-ldflags=-lpthread --with-extra-cflags=-lpthread --with-extra-cxxflags=-lpthread DF=/usr/sysv/bin/df --disable-warnings-as-errors --with-jobs=8'
-    11: '--with-cups-include=/opt/freeware/include --with-extra-ldflags=-lpthread --with-extra-cflags=-lpthread --with-extra-cxxflags=-lpthread DF=/usr/sysv/bin/df --disable-warnings-as-errors --with-jobs=8'
+    11: '--with-cups-include=/opt/freeware/include --disable-warnings-as-errors'
```
Comment (keithc-ca) [inline buildenv/jenkins/variables/defaults.yml]: Perhaps those extra options should be removed for `next` as well?

File: buildenv/jenkins/variables/defaults.yml [hunk 2]
```diff
@@ -146,11 +146,11 @@ aix_ppc-64_cmprssptrs:
...
       next: 'hw.arch.ppc && sw.os.aix'
   extra_configure_options:
-    8: '--with-cups-include=/opt/freeware/include --with-extra-ldflags=-lpthread --with-extra-cflags=-lpthread --with-extra-cxxflags=-lpthread DF=/usr/sysv/bin/df --disable-ccache --with-jobs=8'
-    9: '--with-cups-include=/opt/freeware/include --with-extra-ldflags=-lpthread --with-extra-cflags=-lpthread --with-extra-cxxflags=-lpthread DF=/usr/sysv/bin/df --disable-warnings-as-errors --with-jobs=8'
-    10: '--with-cups-include=/opt/freeware/include --with-extra-ldflags=-lpthread --with-extra-cflags=-lpthread --with-extra-cxxflags=-lpthread DF=/usr/sysv/bin/df --disable-warnings-as-errors --with-jobs=8'
-    11: '--with-cups-include=/opt/freeware/include --with-extra-ldflags=-lpthread --with-extra-cflags=-lpthread --with-extra-cxxflags=-lpthread DF=/usr/sysv/bin/df --disable-warnings-as-errors --with-jobs=8'
-    next: '--with-cups-include=/opt/freeware/include --with-extra-ldflags=-lpthread --with-extra-cflags=-lpthread --with-extra-cxxflags=-lpthread DF=/usr/sysv/bin/df --disable-warnings-as-errors --with-jobs=8'
+    8: '--with-cups-include=/opt/freeware/include --disable-warnings-as-errors'
```
Comment (keithc-ca) [inline buildenv/jenkins/variables/defaults.yml]: Apparently, jdk8 doesn't understand `--disable-warnings-as-errors`.

**Comment by AdamBrousseau** [comment]
Jenkins compile aix jdk8

**Comment by AdamBrousseau** [comment]
8 - https://ci.eclipse.org/openj9/job/PullRequest-Compile-JDK8-aix_ppc-64_cmprssptrs-OpenJ9/30/
9 - https://ci.eclipse.org/openj9/job/PullRequest-Compile-JDK9-aix_ppc-64_cmprssptrs-OpenJ9/16/
10 - https://ci.eclipse.org/openj9/job/PullRequest-Compile-JDK10-aix_ppc-64_cmprssptrs-OpenJ9/57/
11 - https://ci.eclipse.org/openj9/job/PullRequest-Compile-JDK11-aix_ppc-64_cmprssptrs-OpenJ9/5/

Passes all except 10

**Comment by keithc-ca** [comment]
We may need to retry 10; hopefully this is temporary:
> 15:59:27 1506-350 (S) Error writing to intermediate files. There is not enough space in the file system.

**Comment by AdamBrousseau** [comment]
Jenkins compile aix jdk10

**Comment by AdamBrousseau** [comment]
10 passed too
https://ci.eclipse.org/openj9/job/PullRequest-Compile-JDK10-aix_ppc-64_cmprssptrs-OpenJ9/58/

**Comment by AdamBrousseau** [comment]
I'm not sure if we should be removing `--with-jobs=8` as it seems to have slowed down the compiles by about 10 minutes. The configure detects 2 cores to use. I know this has something to do with SMTP but @jdekonin knows a bit more than I do.

**Comment by keithc-ca** [comment]
We should add `--with-jobs=N` only where it appears to be necessary. My experience with Java 11 and 12 suggests it's not necessary for those versions.

**Comment by AdamBrousseau** [comment]
Suggest leaving it in until ibmruntimes/openj9-openjdk-jdk8#57 is resolved.

**Pull request merged**